<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalDeviceManagement</class>
 <widget class="QWidget" name="LocalDeviceManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>MC:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mc">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="objectType"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Instance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="objectInstance">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Property:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="property"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>NoE:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="noe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Index:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="startIndex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>4095</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QCheckBox" name="manualInput">
       <property name="text">
        <string>Manual input</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>cEmi frame (HEX):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_cemiFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>fc000001001000</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_cemiData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deviceManagementSendRequest">
       <property name="text">
        <string>Send Request</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textOuputDeviceManagement"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectRequestDeviceManagement">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectRequestDeviceManagement">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>manualInput</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>mc</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>objectType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>startIndex</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>property</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>noe</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualInput</sender>
   <signal>clicked(bool)</signal>
   <receiver>objectInstance</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
